server/command.go

```diff
@@ -125,6 +127,22 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 }
 
 func (p *Plugin) executeLinkCommand(c *plugin.Context, args *model.CommandArgs, parameters []string) (*model.CommandResponse, *model.AppError) {
+	isSysAdmin, err := p.IsSystemAdmin(args.UserId)
+	if err != nil {
+		return p.cmdError(args.UserId, args.ChannelId, "Something went wrong.")
```

Comment:
should not we pass the actual error here?

Reply:
If not at least log the error

Reply:
No, because the user doesn't need to know the actual error.